tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "cypress/support/component.*",
     "cypress/support/commands.ts"
   ],
+  "exclude": ["src/stories/**"],
```

Comment:
waarvoor is deze eigenlijk?

Reply:
Nadat de laatste adres-crab merge gebeurd was faalde de build op stories.
Ook met changes in deze container stories waren er problemen.

Het faalde op de type-generatie, maar type generatie op storybook stories hebben we niet nodig, vandaar dat die geexclude wordt.